platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/camera/CameraPosition.java

```diff
@@ -168,7 +170,7 @@ public int hashCode() {
    * Builder for composing CameraPosition objects.
    */
   public static final class Builder {
-
+    private boolean needShift = true;
```

Comment:
`needShift` should be `false` by default and only set to `true` if  `Builder#target` is called.